doc/news/_preparation_next_release.md

```diff
@@ -289,8 +289,13 @@ you up to date with the multi-language support provided by Elektra.
 
 - We now use Debian sid to build the documentation instead of Debian stretch. The Doxygen version in Debian stretch [contains a bug](https://github.com/doxygen/doxygen/issues/6456) that causes the generation of the PDF documentation to fail. _(René Schwaiger)_
 - Use Fedora 33 and 32, drop Fedora 31 use in Jenkins. _(Mihael Pranjić)_
-- <<TODO>>
-- <<TODO>>
+- We refactored shared code between pipelines into a [Jenkins Shared Library](https://github.com/ElektraInitiative/jenkins-library). _(Robert Sowula)_
+- The Main and Release Pipeline now creates packages for Debian Buster, Ubuntu Bionic, Ubuntu Focal and Fedora-33. These packages are also installed and automatically tested before they are published. To install these packages, please refer to our [Install documentation](../INSTALL.md). _(Robert Sowula)_
+- We updated our Release Pipeline to push changes directly to our git repositories. _(Robert Sowula)_
+
+### Package Repositories
```

Comment:
You can make a highlight out of this. Add least add link to doc/INSTALL.md or even write here how to install the packages.

Reply:
And please also write about the major version dump of SO and the package names. But also write that in the 0.9.* series the APIs, including the Java binding (which also got a pump here), are still work in progress.